web/src/main/java/org/cbioportal/web/config/CustomObjectMapper.java

```diff
@@ -119,6 +121,7 @@ public CustomObjectMapper() {
         mixinMap.put(Sample.class, SampleMixin.class);
         mixinMap.put(SampleList.class, SampleListMixin.class);
         mixinMap.put(TypeOfCancer.class, TypeOfCancerMixin.class);
+        mixinMap.put(ResourceDefinition.class, ResourceDefinitionMixin.class);
```

Comment:
I just want to double check here whether `ResourceData` needs to be included in this map or not

Reply:
never mind, I realize now that only `ResourceDefinition` has a corresponding `ResourceDefintionMixin` class.